api/src/main/java/org/eclipse/microprofile/metrics/Timer.java

```diff
@@ -100,6 +100,14 @@
     @Override
     long getCount();
 
+    @Override
+    /**
+     * Returns the total elapsed timing durations of all completed timing events that are recorded, in nanoseconds
+     *
+     * @return the elapsed time in nanoseconds
+     */
+    long getSum();
```

Comment:
If this interface extends `Summing` then it contains the `getSum()` method automatically, is there any particular reason to override it here (which has basically no effect AFAIU)? Is it only to be able to add specific javadoc for it?